### PR TITLE
updated to new workshop edition 2023

### PR DESCRIPTION
**Pull request by yannicnoller**
For now, the new version does not show much information, but it puts the 2022 version into the "previous edition" navigation. We can add more info as we get them.

### CODE REVIEW

**Review by lszekeres — APPROVED** [review]
Thanks! 

We probably don't need to keep the kitten pictures :)

File: index.html [hunk 1]
```diff
@@ -37,39 +38,46 @@
...
         empirical analyses, techniques, and applications in the area of fuzzing and 
         software testing for automated bug finding. 
     </p>
     <p>
         The workshop will be organized as Phase 1 in a 2-phase preregistration-based 
-        publication process.  All research papers will be peer-reviewed on the basis of a
+        publication process. All research papers will be peer-reviewed on the basis of a
         full-length preregistered report, and acceptance will be based on (i) the 
         significance and novelty of the hypotheses or techniques, and (ii) the soundness 
         and reproducibility of the methodology specified to validate the claims or
```
Comment (mboehme) [inline index.html:83]: Maybe we can add something about Stage 1, Stage 2, and Artifact Eval at TOSEM.

**Review by mboehme — COMMENTED** [review]
Thanks Yannic. LGMT. Just left a small comment about the collab with TOSEM.

**Comment by yannicnoller** [comment]
@mboehme, I added a commit and refined the overview text, and added more info to Phase 2 and TOSEM Registered Papers. I also added the contact email at the bottom of the page.

**Comment by yannicnoller** [comment]
I also further cleaned the 2022 directory (e.g., removed the kitten images) and did the same for the 2023 edition.

File: index.html [hunk 2]
```diff
@@ -70,19 +70,33 @@
...
+    <p>
+        After incorporating this feedback, 
+        in <b>Phase 2</b>, the authors can submit final versions of the registered reports for 
+        review. Accepted papers will be invited as 
+        <a href="https://dl.acm.org/journal/tosem/registered-papers">ACM TOSEM Registered Papers</a> 
+        (where we guarantee the same set of reviewers). The Phase 2 will be also accompanied by an 
+        <b>artifact evaluation</b>.
+    </p>
+    <p>
+        More details about the benefits of this process can be found in this blog
```
Comment (fuzzingworkshop) [inline index.html:99]: Imo, we can skip the additional revision of the papers at FUZZING. Those changes could be accommodated in the camera-ready. Further changes can be always be requested in Stage 1 of TOSEM. What do you think?

BTW: Both, Phase 1 and 2 will all happen at TOSEM, e.g.,
All papers accepted at the workshop will be invited as Phase 1 [ACM TOSEM Registered Papers](https://dl.acm.org/journal/tosem/registered-papers) where we will maintain an overlapping set of reviewers.

**Comment by yannicnoller** [comment]
Hi, all, I updated the text to clarify the TOSEM submissions. I think the current text is now okay, but of course, we can make more edits and additions in the next few days. I will now merge it and also link it on the ISSTA website.

**Pull request merged**